### DISCUSSION TITLE
Anyone figured how to store data on android/iOS with rust?

### DISCUSSION THREAD

**Discussion by franklinblanco** [fzyzcjy/flutter_rust_bridge#699] [Q&A]
Trying to store data with kv library on android/iOS but obviously getting the error:
<img width="1303" alt="Screen Shot 2022-09-09 at 1 59 12 PM" src="https://user-images.githubusercontent.com/72827992/189414967-f6d60ee5-80ec-4283-bfaf-714e525541df.png">

**Comment by fzyzcjy** [comment]
What about googling it :) Looks like permission errors

**Comment by Desdaemon** [comment]
You might want to check out [`path_provider`](https://pub.dev/packages/path_provider) for this.
